src/Compilers/CSharp/Portable/Binder/Binder_Deconstruct.cs

```diff
@@ -608,6 +608,12 @@ private static void FlattenDeconstructVariables(ArrayBuilder<DeconstructionVaria
 
         internal BoundLocalDeconstructionDeclaration BindDeconstructionDeclarationStatement(DeconstructionDeclarationStatementSyntax node, DiagnosticBag diagnostics)
         {
+            var constKeyword = node.Modifiers.FirstOrDefault(SyntaxKind.ConstKeyword);
+            if (constKeyword != default(SyntaxToken))
```

Comment:
@cston I need it to pass it to the Error() function to get the error to highlight the exact token.